test/PizzeriaTest.cpp

```diff
@@ -65,3 +63,23 @@ TEST_F(PizzeriaTest, calculatePriceForPizzaMock)
 
     delete mock;
 }
+
+TEST_F(PizzeriaTest, orderStubAndTwoMockPizzas) {
+    // Given
+    StrictMock<PizzaMock>* strict_mock = new StrictMock<PizzaMock>{"strict", 1, minutes(0)};
+    NiceMock<PizzaMock>* nice_mock = new NiceMock<PizzaMock>{"nice", 2, minutes(3)};
+    Pizzas pizzas = {new PizzaStub{"STUB"}, strict_mock, nice_mock};
```

Comment:
I think that here will be better create Mocks as a objects and passing them into pizzas by reference :)

Reply:
Yeah, definitely it would be easier